### ISSUE TITLE
[NoiseEngine] example script and python module are broken (Trac #1230)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1224]
Example script `../resources/scripts/example.py` is broken because `I3IsolatedHitsCutModule` does not exist anymore (should probably be replaced by STTools). The same is true for `../python/NoiseEngine.py`. Moreover, the import statements should be checked because some modules are missing and other are not needed. In general, more test scripts are needed.

Migrated from https://code.icecube.wisc.edu/ticket/1230
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Example script `../resources/scripts/example.py` is broken because `I3IsolatedHitsCutModule` does not exist anymore (should probably be replaced by STTools). The same is true for `../python/NoiseEngine.py`. Moreover, the import statements should be checked because some modules are missing and other are not needed. In general, more test scripts are needed.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[NoiseEngine] example script and python module are broken", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-19T21:50:50", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T12:35:39: 
* mjl5147 changed owner from "mlarson" to "mjl5147"
* mjl5147 changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160219T17:04:35: 
* mjl5147 changed resolution from "" to "fixed"
* mjl5147 commented:

> In changeset 142115:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142115"
> Removing the I3DOMLaunch templating (which shouldn't be used anyway: the InIceRawData doesn't exist in 2012+ anyway and the module has always officially been run on I3RecoPulses), added documentation to close #1223, added a couple of python test scripts, and fixed the tray segment to close #1224. Removed the unused serialization from the HitPair class as well
> ```


* mjl5147 changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
